service/src/main/java/org/mskcc/cmo/metadb/service/MetadbMessagingApp.java

```diff
@@ -0,0 +1,78 @@
+package org.mskcc.cmo.metadb.service;
+
+import org.mskcc.cmo.messaging.model.LinkedPatient;
+import org.mskcc.cmo.messaging.model.LinkedSample;
+import org.mskcc.cmo.messaging.model.PatientMetadata;
+import org.mskcc.cmo.messaging.model.SampleMetadataEntity;
+import org.mskcc.cmo.metadb.persistence.GeneralGraphDbRepository;
+import org.mskcc.cmo.metadb.persistence.PatientMetadataRepository;
+import org.mskcc.cmo.metadb.persistence.SampleMetadataRepository;
+
+import java.util.List;
+import junit.framework.Assert;
+import org.mskcc.cmo.messaging.Gateway;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.neo4j.repository.config.EnableNeo4jRepositories;
+
+@EnableNeo4jRepositories(basePackages="org.mskcc.cmo.metadb.persistence")
+@SpringBootApplication(scanBasePackages="org.mskcc.cmo.messaging")
+public class MetadbMessagingApp implements CommandLineRunner {
+    @Autowired
+    private SampleMetadataRepository sampleMetadataRepository;
+    @Autowired
+    private GeneralGraphDbRepository generalGraphDbRepository;
+    @Autowired
+    private PatientMetadataRepository patientMetadataRepository;
+
+    public static void main(String[] args) {
+        SpringApplication.run(MetadbMessagingApp.class, args);
+    }
+
+    @Override
+    public void run(String... args) throws Exception {
+        System.out.println("\n\nCleaning up graphdb...");
+        generalGraphDbRepository.deleteAllFromGraphDb();
```

Comment:
Does this mean, every time we restart this app, the graph database gets wiped out?

Reply:
yeah that's here just for testing right now

Reply:
Sorry, should have added a note stating so